src/detect-krb5-errcode.c

```diff
@@ -156,8 +157,10 @@ static DetectKrb5ErrCodeData *DetectKrb5ErrCodeParse (const char *krb5str)
     krb5d = SCMalloc(sizeof (DetectKrb5ErrCodeData));
     if (unlikely(krb5d == NULL))
         goto error;
-    krb5d->err_code = (int32_t)atoi(arg1);
-
+    if (StringParseInt32(&krb5d->err_code, 10, 0,
```

Comment:
Error codes are [0, 76]. Suggest uint8 range check or parsing as a uint8.

Reply:
Do you know why `err_code` is defined as `int32` in the `krb5d` structure ?

Reply:
Could you please tell where did you check the error codes to be [0, 76]? I could find https://web.mit.edu/kerberos/krb5-1.5/krb5-1.5.4/doc/krb5-admin/Kerberos-V5-Library-Error-Codes.html, is this the wrong place to look at?